pages/common/vue-init.md

```diff
@@ -0,0 +1,17 @@
+# vue init
+
+> Legacy init subcommand for the Vue.js framework.
```

Comment:
How about this?

```suggestion
> Legacy project initialization subcommand from the Vue.js framework.
```

(Or "of the Vue.js framework", if that reads better to you.)

Reply:
Changes applied. Thanks for your suggestions!